src/os/windows/library_loader_win.cpp

```diff
@@ -0,0 +1,61 @@
+/*
+ * This file is part of OpenTTD.
+ * OpenTTD is free software; you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, version 2.
+ * OpenTTD is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details. You should have received a copy of the GNU General Public License along with OpenTTD. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/** @file library_loader_win.cpp Implementation of the LibraryLoader for Windows */
+
+#include "../../stdafx.h"
+
+#include <windows.h>
+
+#include "../../library_loader.h"
+
+#include "../../safeguards.h"
+
+static std::string GetLoadError()
+{
+	auto error = GetLastError();
+	std::string message;
+	char buffer[512];
+	if (FormatMessageA(FORMAT_MESSAGE_FROM_SYSTEM | FORMAT_MESSAGE_IGNORE_INSERTS, NULL, error,
+		MAKELANGID(LANG_NEUTRAL, SUBLANG_DEFAULT), buffer, sizeof(buffer), NULL) == 0) {
```

Comment:
```suggestion
	if (FormatMessageA(FORMAT_MESSAGE_FROM_SYSTEM | FORMAT_MESSAGE_IGNORE_INSERTS, nullptr, error,
		MAKELANGID(LANG_NEUTRAL, SUBLANG_DEFAULT), buffer, sizeof(buffer), nullptr) == 0) {
```
Also, shouldn't we pass the lang ID from the language file here (`_current_language->winlangid`), so we get a translated error message back from Windows?

Reply:
We don't in the other cases we use this function in a nearly identical way (which also use `NULL` btw :P Not for this PR to fix).

I think we talked about it during the development of the network variant of this function, and considered it to be better to use the system language, as then it matches the console language. Not sure, but I believe we spend time on this :)

Either way, if we want this, we should fix all instances of this function; so going to push that out of scope of this PR, if you don't mind :)

Reply:
Since https://github.com/OpenTTD/OpenTTD/pull/11627 that changed; we now use `nullptr` in those cases.